dictionary.md

```diff
@@ -126,9 +146,23 @@ Gherkin is a domain-specific programming language created to write behavior desc
 
 ## H
 
+### Health Check
+
+<!-- TODO -->
```

Comment:
Information about the health of Instance. It informs whether an Instance is running and what is its resource consumption.

It can be accessed directly through Instance API, in the CLI or Panel.

The Instance API health check endpoint is `/health`. The following is an example health check from the Instance obtained with curl:
```
$ curl http://127.0.0.1:8000/api/v1/instance/e9432a4f-2a64-41a2-82f1-39503a2a18df/health |jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   226    0   226    0     0   220k      0 --:--:-- --:--:-- --:--:--  220k
{
  "cpuTotalUsage": 223212751,
  "healthy": true,
  "limit": 536870912,
  "memoryMaxUsage": 14352384,
  "memoryUsage": 12136448,
  "networkRx": 19852,
  "networkTx": 20174,
  "containerId": "df0aa7adf027465da1dfb08edd0a66dc8ffc8e0bd1f5d0fc7fafecbefdac72c6"
}
```

While the below is an example of health check from the same Instance but executed using the CLI interface:
```
$ si inst health e9432a4f-2a64-41a2-82f1-39503a2a18df
Request ok: http://127.0.0.1:8000/api/v1/instance/e9432a4f-2a64-41a2-82f1-39503a2a18df/health status: 200 OK
{
  cpuTotalUsage: 203802417,
  healthy: true,
  limit: 536870912,
  memoryMaxUsage: 14352384,
  memoryUsage: 11980800,
  networkRx: 16266,
  networkTx: 15358,
  containerId: 'df0aa7adf027465da1dfb08edd0a66dc8ffc8e0bd1f5d0fc7fafecbefdac72c6'
}

```

Reply:
added, thank you Alice :)